docker/kafka-setup/kafka-setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+: ${PARTITIONS:=1}
+: ${REPLICATION_FACTOR:=1}
+
+: ${KAFKA_PROPERTIES_SECURITY_PROTOCOL:=PLAINTEXT}
+
+echo "bootstrap.servers=$KAFKA_BOOTSTRAP_SERVER" > ssl.properties
```

Comment:
Can we add the ssl.properties only when KAFKA_PROPERTIES_SECURITY_PROTOCOL is SSL?

Reply:
I renamed the file and added the ssl properties only when the protocol is ssl. But I have to create a properties file, otherwise I would have to duplicate all the kafka-cli commands without the config argument.

Reply:
I think this is good enough. I was previously getting some warning messages, but this should solve it!